package.json

```diff
@@ -75,5 +75,10 @@
   "resolutions": {
     "sass": "^1.54.3"
   },
-  "packageManager": "yarn@3.6.0"
+  "packageManager": "yarn@3.6.0",
+  "dependencies": {
+    "@hookform/resolvers": "^3.1.1",
+    "react-hook-form": "^7.45.1",
+    "zod": "^3.21.4"
+  }
```

Comment:
Can these be added to `devDependencies`?

Reply:
CC: @denniskigen

Reply:
No, these shouldn't need to be devDeps.